example_test.go

```diff
@@ -135,10 +135,14 @@ func ExampleAliasMethods() {
 
 	sql, _, _ = db.From("test").Select(goqu.L("sum(amount)").As("total_amount")).ToSql()
 	fmt.Println(sql)
+
+	sql, _, _ = db.From("test").Select(goqu.I("test.a").As(goqu.I("test.b"))).ToSql()
+	fmt.Println(sql)
 	// Output:
 	// SELECT "a" AS "as_a" FROM "test"
 	// SELECT COUNT(*) AS "count" FROM "test"
 	// SELECT sum(amount) AS "total_amount" FROM "test"
+	// SELECT "test"."a" AS "test"."b" FROM "test"
```

Comment:
I don't think this is valid sql, I tried running the example on postgres and mysql and received syntax errors.

Can you provide an example that would work on all databases like.

```go
db.From("test").Select(goqu.I("test.a").As(goqu.I("b")))
```